pom.xml

```diff
@@ -164,7 +164,7 @@
                                     <encoding>UTF-8</encoding>
                                     <failOnViolation>true</failOnViolation>
                                     <failsOnError>false</failsOnError>
-                                    <logViolationsToConsole>false</logViolationsToConsole>
+                                    <logViolationsToConsole>true</logViolationsToConsole>
```

Comment:
Is this a change from local debugging? I think we decided to not log the checkstyle failures, because its crowding the log too much.

Reply:
Actually yes, but i thought as we have only around 60 violations left in core, the log crowding is acceptable for saving much debugging time by already seeing the voilations at travis build log in case the build is failing because of to much violations.